ozaria/site/components/interactive/PageInteractive/common/BaseInteractiveLayout.vue

```diff
@@ -0,0 +1,82 @@
+<script>
+  import BaseInteractiveTitle from './BaseInteractiveTitle'
+
+  export default {
+    components: {
+      'interactive-title': BaseInteractiveTitle
+    },
+
+    props: {
+      interactive: {
+        type: Object,
+        required: true
+      },
+
+      artUrl: {
+        type: String,
+        default: undefined
+      },
+    }
+  }
+</script>
+
+<template>
+  <div class="interactive-container">
+    <interactive-title
+      :interactive="interactive"
```

Comment:
It seems that this title(prompt text) is going to go into the acodus chrome border, so might need to change later

Reply:
Also, are we using the english text here or i18n localized text? Its ok to not prioritize translation for now, but just curious.

Reply:
`BaseInteractiveTitle` handle the i18n of the title